build.gradle.kts

```diff
@@ -47,9 +48,3 @@ tasks.withType<KotlinCompile> {
 		javaParameters = true
 	}
 }
-
-graphql {
-	schema {
-		packages = listOf("org.misarch.order")
-	}
-}
```

Comment:
What happened to the graphql settings?

Reply:
this is only for a gradle plugin which could generate a GraphQL schema without actually running the service
... which for us did not work for some schema transformation related reasons (probably)
so this did actually nothing